common/thorhelper/thorsoapcall.cpp

```diff
@@ -738,7 +738,7 @@ interface IWSCAsyncFor: public IInterface
     virtual void processException(const Url &url, ConstPointerArray &inputRows, IException *e) = 0;
     virtual void checkTimeLimitExceeded(unsigned * _remainingMS) = 0;
 
-    virtual void createHttpRequest(Url &url, StringBuffer &request) = 0;
+    virtual void createHttpRequest(Url &url, StringBuffer &request, IProperties * traceHeaders) = 0;
```

Comment:
client span declared in function handling soaprequest, and passes clientheaders into the createrequest function, optionally we could set the client span as the "active" span